symplyphysics/laws/chemistry/cross_section_of_interaction_in_elastic_interaction_model.py

```diff
@@ -15,8 +15,8 @@
 cross_sectional_area_of_interaction = Symbol("cross_sectional_area_of_interaction", units.area)
 
 diameter_of_atom = Symbol("diameter_of_atom", units.length)
-constant_sutherland = Symbol("constant_sutherland", units.temperature)
-temperature = Symbol("temperature", units.temperature)
+constant_sutherland = clone_symbol(symbols.thermodynamics.temperature, "constant_sutherland")
+temperature = clone_symbol(symbols.thermodynamics.temperature, "temperature")
```

Comment:
We usually clone, when it has another name, or additional limitations. Here you can simply use global symbol: `temperature = symbols.thermodynamics.temperature`